README.md

```diff
@@ -1,32 +1,22 @@
 Caddy Module: http.handlers.cache
 ================================
 
-**⚠️ Work-in-progress**
-
-This is a distributed HTTP cache module for Caddy.
+This is a distributed HTTP cache module for Caddy based on [Souin](https://github.com/darkweak/souin) cache.
 
 ## Features
 
-* Supports most HTTP cache headers defined in [RFC 7234](https://httpwg.org/specs/rfc7234.html) (see the TODO section for limitations)
-* Sets [the `Cache-Status` HTTP Response Header](https://httpwg.org/http-extensions/draft-ietf-httpbis-cache-header.html)
+ * Supports most HTTP cache headers defined in [RFC 7234](https://httpwg.org/specs/rfc7234.html) (see the TODO section for limitations)
```

Comment:
The TODO doesn't list any missing HTTP cache header support, does it no longer apply and should be removed?

```suggestion
 * Full support for HTTP cache headers defined in [RFC 7234](https://httpwg.org/specs/rfc7234.html).
```

or

```suggestion
 * [RFC 7234](https://httpwg.org/specs/rfc7234.html) compliant HTTP Cache.
```

Reply:
I think the second one is better.